marketdata/stream/conn_coder.go

```diff
@@ -24,11 +24,12 @@ func newCoderWebsocketConn(ctx context.Context, u url.URL) (conn, error) {
 	reqHeader := http.Header{}
 	reqHeader.Set("Content-Type", "application/msgpack")
 	reqHeader.Set("User-Agent", alpaca.Version())
-	//nolint:bodyclose // According to its docs: you never need to close resp.Body yourself
-	conn, _, err := websocket.Dial(ctxWithTimeout, u.String(), &websocket.DialOptions{
+	//nolint:bodyclose // Linter Error
```

Comment:
I actually prefer the original here with the clear nolint explanation. In the new version `ignoredResponse.Body` is actually nil, so calling closeResp on it does nothing.